telegram_upload/management.py

```diff
@@ -150,6 +150,13 @@ def upload(files, to, config, delete_on_success, print_file_id, force_file, forw
         to = async_to_sync(interactive_select_dialog(client))
     elif to is None:
         to = 'me'
+    elif to is not None:
```

Comment:
This code is now unnecessary. Also, it doesn't follow the Python style rules. You should never use except without defining an exception. It is also preferable to validate numbers without using exceptions. The use of defining the variable _ is not necessary.